nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -3657,29 +3657,12 @@ The Provenance Repository contains the information related to Data Provenance. T
 
 |====
 |*Property*|*Description*
-|`nifi.provenance.repository.implementation`|The Provenance Repository implementation. The default value is `org.apache.nifi.provenance.WriteAheadProvenanceRepository`.
-Three additional repositories are available as well.
+|`nifi.provenance.repository.implementation`|The Provenance Repository implementation.The default value is
```

Comment:
```suggestion
|`nifi.provenance.repository.implementation`|The Provenance Repository implementation. The default value is
```

Reply:
Thanks for catching that spacing issue, I pushed a correction.